build-tools/packages/build-cli/src/commands/generate/packageJson.ts

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Flags } from '@oclif/core'
+import { generateMonoRepoInstallPackageJson, MonoRepoKind, Timer } from "@fluidframework/build-tools";
+import { BaseCommand } from '../../base'
+
+export class GeneratePackageJson extends BaseCommand {
+  static description = 'describe the command here'
+
+  static flags = {
+    server: Flags.enum({
+        description: "",
+        options: [ MonoRepoKind.Server ],
+        required: false,
+     }),
+    azure: Flags.enum({
+        description: "",
+        options: [ MonoRepoKind.Azure ],
+        required: false,
+     }),
+    buildTools: Flags.enum({
+        description: "",
+        options: [ MonoRepoKind.BuildTools ],
+        required: false,
+     }),
+     client: Flags.enum({
+        description: "",
+        options: [ MonoRepoKind.Client ],
+        required: false,
+     }),
```

Comment:
This is cool! Do these become unary flags, like `--azure`?

Another option would be to add a `releaseGroup` flag. That would be consistent with the other commands.

Reply:
It seems I was using the enum flag incorrectly. I updated the code -- https://github.com/microsoft/FluidFramework/pull/11280/files#diff-28f4b388a971fc69365fbec438d27fd3aa3caad6e51106ae845989bc9d7f91e0

Reply:
With current enum implementation the command seems to be like -- 
`./bin/dev generate packageJson --monoRepokind=--server`